apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/V2Server.java

```diff
@@ -66,7 +66,7 @@ private void configureServerInfoMetadata() {
 
     // Lightly customize the capability provider to set publisher name.
     ServerCapabilityStatementProvider capabilityStatementProvider =
-        new ServerCapabilityStatementProvider(this);
```

Comment:
Arg constructor is deprecated. Use no args construct.  
ref: https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/hapi/rest/server/ServerCapabilityStatementProvider.html